db/index.js

```diff
@@ -0,0 +1,13 @@
+const ActionService = require("./ActionService");
+const ExpectionService = require("./ExpectionService");
+const ProjectService = require("./ProjectService");
+const StepService = require("./StepService");
+const TestService = require("./TestService");
+
+module.exports = {
```

Comment:
I think this is also a bad pattern. Better to structure the app in terms of features and let the db service with that feature.

Reply:
I don't think that's necessary yet since it's still early. Even if there were various features and feature repositories were required, wouldn't it still be a good idea to have these "table" repositories for those features to use?

Reply:
probably better to talk about this face2face. I just advocating to not structure the app by the different layers and instead group them by functionality or domain.